src/cltk/alphabet/lat.py

```diff
@@ -1,12 +1,471 @@
+    'mea non habet'
+
+    """
+    text = swallow(text, OBELIZED_WORDS)
+    text = swallow(text, OBELIZED_WORD)
+    text = swallow(text, OBELIZED_PLUS_WORDS)
+    return swallow(text, OBELIZED_PLUS_WORD)
+
+
+def disappear_round_brackets(text: str) -> str:
+    """
+    Remove round brackets and keep the text intact
+    :param text: Text with round brackets.
+    :return: Clean text.
+
+    >>> disappear_round_brackets("trib(unus) mil(itum) leg(ionis) III")
+    'tribunus militum legionis III'
+    """
+    text = text.replace("(", "")
+    return text.replace(")", "")
+
+
+def swallow_editorial(text: str) -> str:
+    """
+    Swallow common editorial morks
+    :param text: Text with editorial marks
+    :return: Clean text.
+
+    >>> swallow_editorial("{PRO P. QVINCTIO ORATIO} Quae res in civitate trib(unus) mil(itum) leg(ionis) III tu Fauonium †asinium† dicas meam +similitudinem+  mea +ratio non habet ...     ")
+    '{PRO P. QVINCTIO ORATIO} Quae res in civitate tribunus militum legionis III tu Fauonium  dicas meam   mea non habet ...'
+
+    """
+    text = disappear_round_brackets(text)
+    text = swallow_angle_brackets(text)
+    text = swallow_square_brackets(text)
+    text = swallow_obelized_words(text)
+    return text
+
+
+def accept_editorial(text: str) -> str:
+    """
+    Accept common editorial suggestions
+    :param text: Text with editorial suggestions
+    :return: clean text
+
+    >>> accept_editorial("{PRO P. QVINCTIO ORATIO} Quae res in civitate trib(unus) mil(itum) leg(ionis) III tu Fauonium †asinium† dicas meam +similitudinem+  mea +ratio non habet ...     ")
+    'Quae res in civitate tribunus militum legionis III tu Fauonium  dicas meam   mea non habet  '
+
+    """
+    text = swallow_braces(text)
+    text = disappear_round_brackets(text)
+    text = swallow_obelized_words(text)
+    text = text.replace("[", "")
+    text = text.replace("]", "")
+    text = text.replace("<", "")
+    text = text.replace(">", "")
+    text = text.replace("...", " ")
+    return text
+
+
+def truecase(word: str, case_counter: Dict[str, int]):
+    """
+    Truecase a word using a Truecase dictionary
+
+    :param word: a word
+    :param case_counter: A counter; a dictionary of words/tokens and their relative frequency counts
+    :return: the truecased word
+
+    >>> case_counts ={"caesar": 1, "Caesar": 99}
+    >>> truecase('CAESAR', case_counts)
+    'Caesar'
+
+    """
+    lcount = case_counter.get(word.lower(), 0)
+    ucount = case_counter.get(word.upper(), 0)
+    tcount = case_counter.get(word.title(), 0)
+    if lcount == 0 and ucount == 0 and tcount == 0:
+        return word  #: we don't have enough information to change the case
+    if tcount > ucount and tcount > lcount:
+        return word.title()
+    if lcount > tcount and lcount > ucount:
+        return word.lower()
+    if ucount > tcount and ucount > lcount:
+        return word.upper()
+    return word
+
+
+def normalize_lat(
```

Comment:
I am surprised we are not using `clkt_normalize`: https://github.com/cltk/cltk/blob/57879c3c54362d194caf66496384449875f0b1c1/src/cltk/alphabet/text_normalization.py#L6

One goal I have had is to run ALL text through this, before training and before inference of user data. Are you open to the idea? Is it really silly to do it for Latin? Greek we need to and more exotic alphabets I assume it'll help. Would like to learn about this.

Reply:
Yes, I would like to run all the text through this prior to inference.
On a higher level, I think we can/need to provide some flowchart guidance, e.g.:
Preprocesing text? 
* `dehyphenate` before tokenizing

Building a custom word embedding?  
* after tokenizing, `drop_latin_punctuation` (don't swallow enclitic hyphens!)

Building a custom Transformers model?
* (don't drop any punctuation, because model can use it)

Reply:
@todd-cook I have resolved all comments except this one. Let's talk here or offline about the pros/cons.

Reply:
>  a higher level, I think we can/need to provide some flowchart guidance

Yeah, this needs to be documented very carefully. Esp as you say different preprocessing for different texts.

ATM the biggest concern I have is which to make a generally *safe* default. It'd be terrible if someone put in fairly normal text and it broken or did something crazy.